conans/client/tools/scm.py

```diff
@@ -175,6 +175,14 @@ def get_commit(self):
 
     get_revision = get_commit
 
+    def get_commit_message(self):
+        self.check_repo()
+        try:
+            message = self.run("log -1 --format=%s%n%b")
+            return message.strip()
```

Comment:
It is already _stripped_, no need to do it again

Reply:
do you mean self.run is stripped? why are there more self.run().strip() declared on SVN class?

Reply:
Sorry, my fault. You are absolutely right. In fact `SVN` class has its own runner called `runner_no_strip` 🤦‍♂️